engine/evarutil.ml

```diff
@@ -453,7 +454,8 @@ let new_evar ?src ?filter ?abstract_arguments ?candidates ?naming ?typeclass_can
     | None -> instance
     | Some filter -> Filter.filter_list filter instance in
   let identity = if Int.equal (Environ.nb_rel env) 0 then Some (Identity.make instance) else None in
-  let (evd, evk) = new_pure_evar sign evd typ' ?src ?filter ?identity ?abstract_arguments ?candidates ?naming
+  let relevance = Sorts.Relevant in (* FIXME: relevant_of_type not defined yet *)
```

Comment:
What's to be done here?

Reply:
Compute the relevance out of the type of conclusion of the evar.